src/Cpp/1-getting-started/12-HelloGLFWWindow/Main.cpp

```diff
@@ -0,0 +1,37 @@
+#include <GLFW/glfw3.h>
+
+#include <iostream>
+
+static void CenterWindow(GLFWwindow* windowHandle)
+{
+    const GLFWvidmode* videoMode = glfwGetVideoMode(glfwGetPrimaryMonitor());
+    int32_t currentWidth = 0;
+    int32_t currentHeight = 0;
+    glfwGetFramebufferSize(windowHandle, &currentWidth, &currentHeight);
+    glfwSetWindowPos(
+        windowHandle,
+        videoMode->width / 2 - currentWidth / 2,
+        videoMode->height / 2 - currentHeight / 2);
+}
+
+int main()
+{
+    const int32_t windowWidth = 1280;
+    const int32_t windowHeight = 720;
```

Comment:
Please do not hardcore any resolution, retrieve the current desktop resolution and use 90% of it instead

Reply:
Understood, I have fixed the issue along with other issues

Reply:
Yeah like the markdown variant it will be taken care of